client/packages/common/src/intl/utils/IntlUtils.ts

```diff
@@ -49,12 +50,12 @@ export const useIntlUtils = () => {
     }
 
     // Handle languages such as en-US or fr-FR
-    const baseLanguage = supportedLanguage.split('-')[0] as SupportedLocales;
+    const baseLanguage = supportedLanguage?.split('-')[0] as SupportedLocales;
```

Comment:
this and the one below are partly because the context is initially empty and this will cause a problem on the first render.